### DISCUSSION TITLE
Remove messages already read on remote device

### DISCUSSION THREAD

**Discussion by riban-bw** [InfiniTimeOrg/InfiniTime#1981] [Ideas]
When a device, e.g. phone sends a message to the watch it is stored on the watch ready to be read. A notification may inform the user of the message and a queue of messages is created ready to be read. There is already an option to avoid messages being sent if the device (phone) is awake, presumably because the user is likely to see the notification on that device and hence not require notification on the watch.

What I would like is for any messages that are consequently processed by the user on the device (phone) are removed from the watch message queue. I often have several notifcation messages on the watch for things that I have long since read and dismissed on the phone.

I don't know if there is an option for notifying the watch through the companion app of this. I do see the watch registers for unread messages so it feels like there should be some mechanism to be able to implement this feature.

**Comment by shade-belisar** [comment]
I would second this proposal. Ideally I would like to see a 2-way sync of notifications. Right now I have to dismiss notifications both on my phone and my watch, which is slightly annoying.

**Reply by gregdotca** [reply]
I really wish I knew this was a thing before I ordered a PineTime and a dev kit, lol. Lesson learned I guess. I've never owned a smart watch that didn't synchronize notifications, and it literally didn't even cross my mind that this would be a thing. I'm mostly just sad that there's no chance I can use the PineTime as my daily driver.

**Comment by stephen-m-shubin** [comment]
I third this proposal.
I recently destroyed my Pebble Time, and I'm using PineTime/Inifinitime as my replacement. For the most part it's doing well, but not having synced message alerts between the watch and phone makes it very annoying to use daily.

**Comment by gregdotca** [comment]
I received my PineTime today and I'm absolutely loving it, but unfortunately not having notifications sync between the watch and companion phone is a deal breaker for me, so I wanted to come add a +1 to this issue. I wish I'd known this was a thing before ordering a dev kit along with a closed watch, but I had high hopes and wanted to save on shipping to Canada, lol.

It's too bad, my first impressions of the PineTime were amazing, and I had some brief hope that I could make it my daily driver, but having to clear notifications in both places kind of defeats the purpose for me, which is saving time.
